### PR TITLE
Added #break directive

### PR DESCRIPTION
**Pull request by mholo65**
Solves #869

### CODE REVIEW

File: src/Cake.Core/Scripting/Processors/BreakDirectiveProcessor.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            }
+
+            replacement = null;
+
+            if (!line.Trim().Equals("#break", StringComparison.Ordinal))
+            {
+                return false;
+            }
+
+            replacement = @"if (System.Diagnostics.Debugger.IsAttached) { System.Diagnostics.Debugger.Break(); }";
```
Comment (mholo65) [inline src/Cake.Core/Scripting/Processors/BreakDirectiveProcessor.cs]: This could be done more elegantly once #902 is implemented

**Comment by patriksvensson** [comment]
@mholo65 Looks good to me! Will merge as soon as I have some spare time later tonight!

**Comment by patriksvensson** [comment]
@mholo65 Sorry to bother you, but could you rebase against upstream/develop and squash so I can merge this. Sorry for the inconvenience.

**Comment by mholo65** [comment]
@patriksvensson rebased and CI passes.

**Comment by patriksvensson** [comment]
Merged! Great work! :+1: :smile:

**Pull request merged**